go.mod

```diff
@@ -4,11 +4,13 @@ go 1.17
 
 require (
 	github.com/docker/docker v20.10.8+incompatible
+	github.com/go-gomail/gomail v0.0.0-20160411212932-81ebce5c23df
```

Comment:
no entirely sure if updating this is ok? just realized this module is updated along the way. If you don't want this update I can remove this.

Reply:
It's still the same version so I guess it's fine. Not sure why it ever has been `indirect` tbh as it's used by the script itself.